libs/components/split-view/src/lib/modules/split-view/split-view-workspace-header.component.ts

```diff
@@ -23,37 +23,42 @@ import { SkySplitViewService } from './split-view.service';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class SkySplitViewWorkspaceHeaderComponent implements OnDestroy, OnInit {
-  public backButtonText: string;
+  public backButtonText: string | undefined;
 
-  private ngUnsubscribe = new Subject<void>();
+  #ngUnsubscribe = new Subject<void>();
+  #splitViewSvc: SkySplitViewService;
+  #changeRef: ChangeDetectorRef;
 
   constructor(
-    private splitViewService: SkySplitViewService,
-    private changeRef: ChangeDetectorRef,
+    splitViewSvc: SkySplitViewService,
+    changeRef: ChangeDetectorRef,
     @Optional() skyThemeSvc?: SkyThemeService
   ) {
+    this.#splitViewSvc = splitViewSvc;
+    this.#changeRef = changeRef;
+
     /*istanbul ignore else*/
     if (skyThemeSvc) {
       skyThemeSvc.settingsChange.subscribe(() => {
-        this.changeRef.markForCheck();
+        this.#changeRef.markForCheck();
```

Comment:
Do you know if subscribing to the theme change is doing anything?

Reply:
This doesn't seem to be doing anything, and the [commit that introduced it](https://github.com/blackbaud/skyux-split-view/commit/506dc3e40d1d8cbe84598d881ef6cab2c16854fe) doesn't really give a clue as to why it's needed. I'm going to remove it and we'll put it back if it causes a problem.

Reply:
Fixed.